DataBase/dataBase.py

```diff
@@ -56,45 +57,23 @@ def store_alert_data(alert_time, video_link, alert_class, alert_type):
         except Exception as e:
             print(f"Error inserting alert into {alert_type}_alerts: {e}")
 
-def retrieve_fire_alerts():
-
-    # Establishing Connection to DB
-    with get_db_connection() as (cursor, conn):
-        query = "select * from fire_alerts"
-        cursor.execute(query)
-        print("Fire alerts : \n ------------------------------------ \n")
-        fire_alerts = cursor.fetchall()
-        for row in fire_alerts:
-            print(f"ID : {row[0]} | alert time : {row[1]} | video link : {row[2]} | class : {row[3]}")
-
-
-def retrieve_fall_alerts():
-
-    # Establishing Connection to DB
-    with get_db_connection() as (cursor, conn):
-        query = "select * from fall_alerts"
-        cursor.execute(query)
-        print("Fall alerts : \n ------------------------------------ \n")
-        fall_alerts = cursor.fetchall()
-        for row in fall_alerts:
-            print(f"ID : {row[0]} | alert time : {row[1]} | video link : {row[2]} | class : {row[3]}")
-
-def retrieve_robbery_alerts():
```

Comment:
leveraging code_reusebility is important here as a centralized function with an additional parameter to decide which table to handle is a better approach